.github/workflows/httpx.yml

```diff
@@ -0,0 +1,36 @@
+name: httpx-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  httpx-testsuite-0_18_1:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: httpx
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: encode/httpx
+          ref: 0.18.1
+          path: httpx
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+      - name: Inject ddtrace
+        run: pip install ../ddtrace
+      - name: Add pytest configuration for ddtrace
+        run: echo -e "[pytest]\nddtrace-patch-all = 1" > pytest.ini
+      - name: Run tests
+        env:
+          # Disabled distributed tracing since there are a lot of tests that assert on headers
+          DD_HTTPX_DISTRIBUTED_TRACING: "false"
+        run: pytest
+        # run: pytest -k "not test_asgi_headers and not test_raw_client_header and not test_client_header and not test_header_merge and not test_header_update and not test_remove_default_header and not test_event_hooks and not test_async_event_hooks and not test_event_hooks_with_redirect and not test_async_event_hooks_with_redirect and not test_host_with_auth_and_port_in_url and not test_host_with_non_default_port_in_url"
```

Comment:
Seems like this commented out line can be safely removed?

Reply:
yes!
```suggestion
```